src/app/design/guidelines/back-button/index.html

```diff
@@ -0,0 +1,71 @@
+<stache
+  pageTitle="Back Button"
+  navTitle="Back button"
+  showTableOfContents="true">
+
+  <stache-page-anchor>
+    New page behavior
+  </stache-page-anchor>
+
+  <p>
+    When users navigate to a new context such as a new record, list or section, the action should create an entry in the browser history with a new page title. This entry enables users to select the browser's back button to return to the previous page or view.
+  </p>
+  <p>
+    As a rule of thumb, create a new page in the browser history if the majority of the page content changes or users perceive a signficant change of context. Don't create a new URL or new browser entry for an inline interaction such as steps in a wizard or opening a detail panel.
```

Comment:
should "a new page" be "an entry"?

Also, I don't think we need "opening" near the end.